src/main.cpp

```diff
@@ -282,6 +282,14 @@ void setup()
     delay(100);
 #endif
 
+#if defined(CROWPANEL_ESP32S3_5_EPAPER)
```

Comment:
We have a LOT of defines already switching GPIO's around. Can you hijack one of the existing defines for these? Would save us a few bytes of cache.

Reply:
let me see

Reply:
@caveman99 ok i have removed the power enable stuff from main and used the other defined enable pins ie VEXT_ENABLE  and PIN_POWER_EN, defined in the variant